tests/output-module-formats/subtests/es-modules/parcel/index.md

```diff
@@ -1,5 +1,14 @@
 ---
-result: pass
+result: fail
+issue:
+  - url: https://github.com/parcel-bundler/website/issues/631
+    status: closed
+  - url: https://github.com/parcel-bundler/website/issues/630
+  - url: https://github.com/parcel-bundler/parcel/issues/4890
 ---
 
-Parcel can build for different “targets”, which are defined in `package.json`. Per target you can set the `outputFormat`, which can be set to `esmodule`. Chunks will be saved as ES modules, but it will only use dynamic import if your browser support list only contains browsers that support dynamic import. Otherwise, it will use `<script>`-based loading instead.
+Parcel lets you set [targets](https://v2.parceljs.org/configuration/package-json/#targets) in your `package.json` to influence how your project is built, including an 'esmodule' setting.
+
+This will change the target browsers to the ones that support ECMAScript modules, but that doesn't include browsers that support dynamic `import()`. By default Parcel will include a polyfill, unless you set the [`browserslist`](https://v2.parceljs.org/configuration/package-json/#engines-%2F-browserslist) to browsers that support `import()`.
```

Comment:
It's nitpicky, but I'm not sure how big of a deal it is to include a dynamic import polyfill given the small size and usefulness of it. Without that they'd have to use some non-static heuristic in place of module/nomodule, which would be quite a drawback.

Reply:
It seems weird to have to change the browserslist if you're building a library for node.

Reply:
You mean when building a (browser) library with an ESM output format? That is indeed not ideal

Reply:
@mischnic yeah. It's possible I was holding it wrong.

Reply:
Seems like you could just use the engines field:

```js
{
  "engines": {
    "node": ">= 12",
    "browsers": "Chrome >= 61"
  }
}
```